manager/apis/app/v1alpha1/ifdetails.go

```diff
@@ -3,40 +3,24 @@
 
 package v1alpha1
 
-// IFProtocol defines interface protocol for data transactions
-// +kubebuilder:validation:Enum=s3;kafka;jdbc-db2;m4d-arrow-flight
-type IFProtocol string
-
-// DataFormatType defines data format type
-// +kubebuilder:validation:Enum=parquet;table;csv;json;avro;orc;binary;arrow
-type DataFormatType string
-
-// DataFormatType valid values
-const (
-	Parquet DataFormatType = "parquet"
-	Table   DataFormatType = "table" // remove?
-	CSV     DataFormatType = "csv"
-	JSON    DataFormatType = "json"
-	AVRO    DataFormatType = "avro"
-	ORC     DataFormatType = "orc"
-	Binary  DataFormatType = "binary"
-	Arrow   DataFormatType = "arrow"
-)
-
-// IFProtocol valid values
+// Values used in tests and mockup servers
```

Comment:
I  think it would be better to move these out of `apis`. Do you have a good suggestion where to?

Reply:
OK It's still used in `CatalogDatasetToDataDetails` so not just for testing until we get rid of the GRPC stuff.